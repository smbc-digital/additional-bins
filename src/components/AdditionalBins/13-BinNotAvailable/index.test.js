import React from 'react'
import Enzyme from 'enzyme'
import Adapter from 'enzyme-adapter-react-16'
import renderer from 'react-test-renderer'
import BinNotAvailable from './index'

Enzyme.configure({ adapter: new Adapter() })

describe('BinNotAvailable', () => {

    describe('snapshot', () => {
        it('renders correctly', () => {

            const tree = renderer
                .create(<BinNotAvailable />)
                .toJSON()
                
            expect(tree).toMatchSnapshot()
        })
    })
})