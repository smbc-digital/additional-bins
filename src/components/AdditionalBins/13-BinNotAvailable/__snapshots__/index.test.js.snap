// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`BinNotAvailable snapshot renders correctly 1`] = `
<form>
  <div
    className="ineligible-warning"
  >
    <div
      className="header-container"
    >
      <div>
        <i
          aria-hidden="true"
          className="fa fa-exclamation"
        />
      </div>
      <h1>
         We're sorry but you can't use this form
      </h1>
    </div>
    <h2>
      Why not?
    </h2>
    <p>
      You must live in a property that already has a black wheelie bin.
    </p>
    <p>
      To be eligible for an additional black wheelie bin, you must have enough outside space to store it off the road. Properties that are not eligible include; flats above shops or terrace properties without a garden.
    </p>
    <h2>
      What you can do next
    </h2>
    <p>
      Please call us on 
      <strong>
        0161 217 6111
      </strong>
       if you think you're still eligible for an additional black wheelie bin or you don't have enough space for your current non-recyclable waste.
    </p>
    <a
      className="button-primary"
      href="https://www.stockport.gov.uk"
    >
      Go to the homepage
    </a>
  </div>
</form>
`;
